membership-attribute-service/app/services/ScanamoAttributeService.scala

```diff
@@ -59,6 +59,8 @@ class ScanamoAttributeService(client: AmazonDynamoDBAsync, table: String)(implic
       scanamoSetOpt('RecurringContributionPaymentPlan -> attributes.RecurringContributionPaymentPlan),
       scanamoSetOpt('MembershipJoinDate -> attributes.MembershipJoinDate),
```

Comment:
I have also noticed that if someone cancels their membership, it is never removed from the cache, because although None == "no membership", in this context None is interpreted as "don't update this field in dynamo" so it is never removed.  This may cause intermittent supporter asks for people who used to have a membership and don't have other products.